ui/components/app/modals/qr-scanner/index.scss

```diff
@@ -34,6 +34,23 @@
     }
   }
 
+  &__progress {
+    width: 110px;
+    height: 4px;
+    border-radius: 2px;
+    overflow: hidden;
+    background-color: #dadada;
```

Comment:
We should not be using any static hex values in the extension this may break when changing between themes, would it be possible to use `var(--color-border-muted)` here instead?

Reply:
Thanks very much. It should be replaced. I didn't notice.